src/operator/softmax_activation.cc

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2015 by Contributors
+ * \file activation.cc
+ * \brief softmax_activation op
+ * \author Junyuan Xie
+*/
+#include "./softmax_activation-inl.h"
+#include "./mshadow_op.h"
+
+namespace mxnet {
+namespace op {
+template<>
+Operator *CreateOp<cpu>(SoftmaxActivationParam param) {
+  LOG(FATAL) << "Softmax Activation for internal layers is only supported "
+                "on GPU with cuDNN. Use SoftmaxOutput for loss layer.";
+  return new SoftmaxActivationOp<cpu>(param);
+}
+
+// DO_BIND_DISPATCH comes from operator_common.h
+Operator *SoftmaxActivationProp::CreateOperator(Context ctx) const {
+  DO_BIND_DISPATCH(CreateOp, param_);
+}
+
+DMLC_REGISTER_PARAMETER(SoftmaxActivationParam);
+
+MXNET_REGISTER_OP_PROPERTY(SoftmaxActivation, SoftmaxActivationProp)
```

Comment:
transform isn't used anywhere else in the code base. It's meaning is kind of unclear